hudi-common/src/main/java/org/apache/hudi/common/util/ParquetReaderIterator.java

```diff
@@ -36,24 +35,20 @@
   private final ParquetReader<T> parquetReader;
   // Holds the next entry returned by the parquet reader
   private T next;
-  // For directly use InternalRow
-  private Function<T, T> mapper;
+  // Whether next is consumed
+  private boolean consumed;
```

Comment:
Why are we changing this one?

Reply:
ParquetReaderIterator#next will return the current row and get the next row. We should get rid of it if we do not want to copy row in ParquetReaderIterator.
```java
  T retVal = this.next;
  // This is what I want to remove if we not copy the retVal.
  this.next = read();
  return retVal;
```